docs/manifest.json

```diff
@@ -230,9 +230,9 @@
           "icon_path": "./images/icons/docker.svg"
         },
         {
-          "title": "Devcontainers",
-          "description": "Use devcontainers in workspaces",
-          "path": "./templates/devcontainers.md",
+          "title": "Dev Containers",
+          "description": "Use Dev Containers in workspaces",
+          "path": "./templates/dev-containers.md",
```

Comment:
nit: could be a separate change but I'm fine with it sneaking in here

Reply:
Yeah, this is a cleanup 👍 Let's leave it here as it brings less "paperwork".